src/main/java/de/retest/recheck/cli/subcommands/Diff.java

```diff
@@ -36,13 +45,28 @@ public void run() {
 		}
 		try {
 			final TestReport report = TestReportUtil.load( testReport );
-			final TestReportPrinter printer = new TestReportPrinter( none(), loadRecheckIgnore() );
+			final TestReportPrinter printer = new TestReportPrinter( none(), checkFilterNames() );
 			logger.info( "\n{}", printer.toString( report ) );
 		} catch ( final IOException e ) {
 			logger.error( "Differences couldn't be printed:", e );
 		}
 	}
 
+	CompoundFilter checkFilterNames() throws IOException {
+		final List<Filter> filters = new ArrayList<>();
+		final CompoundFilter compoundFilter = new CompoundFilter( filters );
```

Comment:
You do not need the `CompoundFilter` here. Please make the variable scope as small as possible and only create it in `return`. That would also prevent the mutability of the `CompoundFilter`, if someone decided to do the `CompoundFilter` properly by copying the list instead of just referencing it.